File: Controle Arduino Desktop/Controle Arduino Desktop/Controller/BuzzerController.vb
Public Class BuzzerController

    Public Shared Function TocarBuzzerAlarme()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/tocar/alarme")

        Return resposta


    End Function
    Public Shared Function PausarBuzzerAlarme()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/pausar/alarme")

        Return resposta


    End Function
    Public Shared Function DesligarBuzzerAlarme()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/desligar/alarme")

        Return resposta


    End Function
    Public Shared Function StatusBuzzerAlarme()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/status")

        Return resposta


    End Function
    Public Shared Function TocarBuzzerAmbulancia()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/tocar/ambulancia")

        Return resposta


    End Function
    Public Shared Function PausarBuzzerAmbulancia()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/pausar/ambulancia")

        Return resposta


    End Function
    Public Shared Function DesligarBuzzerAmbulancia()
        Dim netApi = New ConnectionApi()

        Dim resposta As Buzzer = netApi.BuzzerApi("http://localhost:8080/buzzer/desligar/ambulancia")

        Return resposta


    End Function

End Class

File: Controle Arduino Desktop C-SHARP/Controle Arduino Desktop C-SHARP/Controller/BuzzerController.cs
using Controle_Arduino_Desktop_C_SHARP.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Controle_Arduino_Desktop_C_SHARP.Controller
{
    class BuzzerController
    {

        public Buzzer TocarBuzzerAlarme() {
            var netApi = new ConnectionApi();

            var resposta = (Buzzer)netApi.BuzzerApi("http://localhost:8080/buzzer/tocar/alarme");

            return resposta;


    }
        public Buzzer PausarBuzzerAlarme() {

            var netApi = new ConnectionApi();
    
        var resposta = (Buzzer) netApi.BuzzerApi("http://localhost:8080/buzzer/pausar/alarme");

        return resposta;
    

    }

    public Buzzer DesligarBuzzerAlarme() {
            var netApi = new ConnectionApi();

            var resposta = (Buzzer)netApi.BuzzerApi("http://localhost:8080/buzzer/desligar/alarme");

            return resposta;


    }

    public Buzzer StatusBuzzerAlarme() {
            var netApi = new ConnectionApi();

            var resposta = (Buzzer)netApi.BuzzerApi("http://localhost:8080/buzzer/status");

            return resposta;


    }

    public Buzzer TocarBuzzerAmbulancia() {
        
            var netApi = new ConnectionApi();

            var resposta = (Buzzer) netApi.BuzzerApi("http://localhost:8080/buzzer/tocar/ambulancia");

            return resposta;


    }

    public Buzzer PausarBuzzerAmbulancia() {
            var netApi = new ConnectionApi();

            var resposta = (Buzzer)netApi.BuzzerApi("http://localhost:8080/buzzer/pausar/ambulancia");

            return resposta;


    }

    public Buzzer DesligarBuzzerAmbulancia() {
            var netApi = new ConnectionApi();

            var resposta = (Buzzer) netApi.BuzzerApi("http://localhost:8080/buzzer/desligar/ambulancia");

            return resposta;


    }

    }
}
